haproxy/datadog_checks/haproxy/haproxy.py

```diff
@@ -193,17 +215,31 @@ def _fetch_socket_data(self, parsed_url):
         else:
             sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
             sock.connect(parsed_url.path)
-        sock.send(b"show stat\r\n")
+        # returns both version info and stats data
+        sock.send(b"show info;show stat\r\n")
 
         response = ""
         output = sock.recv(BUFSIZE)
         while output:
             response += output.decode("ASCII")
             output = sock.recv(BUFSIZE)
-
         sock.close()
-
-        return response.splitlines()
+        # return data from `show info` and `show stat` separately
+        # getting the first 2 values as the third one is empty string
+        return response.split('\n\n')[:2]
```

Comment:
```suggestion
        info, data = response.split('\n\n')[:2]
        return info, data.splitlines()
```
And then remove `data = data.splitlines()` from https://github.com/DataDog/integrations-core/pull/4851/files#diff-ab68ac5f4214da2f3476b029ac27164dR109